<launch>
  <arg name="manager" default="color_detector_manager" />
  <arg name="standalone" default="false" />
  <arg name="image" />

  <node if="$(arg standalone)" name="$(arg manager)" pkg="nodelet" type="nodelet" args="manager" output="screen" />

  <node name="color_detector" pkg="nodelet" type="nodelet" args="load object_detection/ColorDetector $(arg manager)" output="screen" >
    <remap from="color_detector/image" to="$(arg image)" />
    <rosparam file="$(find object_detection)/cfg/color_detector_params.yaml" />
    <param name="db_type" value="filesystem" />
    <param name="connection_string" value="$(find object_detection)/models" />
  </node>

</launch>
